index.html

```diff
@@ -88,8 +91,8 @@
     <h1>Year Progress</h1>
     <img id="progress-img" src="https://year-in-progress.vercel.app/year-progress" alt="Year Progress">
 
-    <!-- Button positioned directly below the progress bar and aligned to the left -->
-    <button id="theme-toggle" class="btn"><i>Switch to Dark Mode</i></button>
+    <!-- Button positioned directly below the progress bar and aligned to the left --> icon
```

Comment:
Why this extra text `icon` showing??  Please remove it 
![Screenshot from 2024-10-09 19-02-14](https://github.com/user-attachments/assets/fa4cb18c-20de-4400-a8e1-31263096b631)